src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.wallet.account-overview.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]
+            [quo2.components.wallet.account-overview.style :as style]))
+
+(defn account-overview
+  [{:keys [account-name current-value bottom-time-text bottom-time-to-text show-details?
+           percentage-change currency-change up?]}]
+  [rn/view
+   {:style style/account-overview-wrapper}
+   [rn/view style/row
```

Comment:
can you instead do `[rn/view {:style style/row} ...` ?

Reply:
Done